app/assets/stylesheets/application.css.scss

```diff
@@ -26,16 +26,22 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 
 
 .post {
-  border-top: solid black 1px;
+  border-radius: 10px;
```

Comment:
Properties should be ordered background-color, border-radius, list-style, margin, padding